config/prod.exs

```diff
@@ -10,8 +10,8 @@ use Mix.Config
 # which you should run after static files are built and
 # before starting your production server.
 config :project_tides, ProjectTidesWeb.Endpoint,
-  url: [host: "example.com", port: 80],
-  cache_static_manifest: "priv/static/cache_manifest.json"
+  url: [host: System.get_env("BASE_URL"), port: 80]
+  # cache_static_manifest: "priv/static/cache_manifest.json"
```

Comment:
Did you comment out the line as we don't intend to serve any static content? Or some other reason?

Reply:
That's correct @rudresh we are targetting for `API` driven app. We'll not be compiling any of the static assets. Removing this line of code.